### PR TITLE
[zig/en] Remove incorrect comment in enum section

### PR DESCRIPTION
**Pull request by chupsondev**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]` (example `[python/fr-fr]` or `[java/en]`)
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [ ] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [ ] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [ ] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: zig.html.markdown [hunk 1]
```diff
@@ -588,9 +588,6 @@ const x = switch (direction) {
     .South => true,
 };
 
-
-// Switch statements need exhaustiveness.
-// Won't compile: East and West are missing.
 const x = switch (direction) {
```
Comment (vendethiel) [inline zig.html.markdown:591]: Can you add another comment instead saying how that one's fine since it's exhaustive? Thanks

Comment (chupsondev) [reply zig.html.markdown:591]: Sure, sounds good

Comment (chupsondev) [reply zig.html.markdown:592]: Solved with 5cb021cf3e53373770c598062b8079b6d23c9e64

**Comment by vendethiel** [comment]
Thank you!

**Pull request merged**